src/main/java/com/google/jepsenonspanner/verifier/LinearizabilityVerifier.java

```diff
@@ -11,7 +15,20 @@ public boolean verify(Map<String, Long> initialState, String... filePath) {
     if (filePath.length != 2) {
       throw new RuntimeException("Linearizability Verifier only accepts 2 files");
     }
-    return new KnossosVerifier().verify(initialState, filePath[0]) && new ExternalConsistencyVerifier().verify(
-            initialState, filePath[1]);
+
+    return timedVerify(new KnossosVerifier(), /*filePathIdx=*/0, initialState, filePath) &&
+            timedVerify(new ExternalConsistencyVerifier(), /*filePathIdx=*/1, initialState,
+                    filePath) &&
+            timedVerify(new LinearVerifier(), /*filePathIdx=*/0, initialState, filePath);
```

Comment:
Update the comment to demonstrate that we are running both Knossos and our WGL verifier.